docs/docs/noir/standard_library/options.md

```diff
@@ -56,6 +56,10 @@ Returns the wrapped value if `self.is_some()`. Otherwise, returns the given defa
 
 Returns the wrapped value if `self.is_some()`. Otherwise, calls the given function to return a default value.
 
+### expect
+
+Asserts `self.is_some()` with a provided custom message and returns the contained `Some` value. The custom message is expected to be a format string.
```

Comment:
Making a note of this: we should add examples to the documentation of each of these functions in a later PR